images/plbase/python-requirements.txt

```diff
@@ -28,3 +27,4 @@ openpyxl==3.0.10
 pytest==7.2.0
 typing-extensions==4.4.0
 psutil==5.9.4
+black==22.10.0
```

Comment:
👀 did you mean to invoke Black somewhere, perhaps in the Makefile?

Reply:
So the more I think about it, I should probably remove this line until the backend has been black-ified. My original intent was to have it installed so people can run it if they choose and know what they're doing, but this actually doesn't seem like a good idea.

I have another PR where I actually added this to the makefile. The issue is, adding that command to the makefile runs it over the whole codebase, and that leads to a huge number of changes. I don't think we want people going out of their way to run it like that until it's been run over the whole backend, and running it doesn't lead to a giant diff.

Reply:
Yep - let's hold off on installing Black or running it on any files until that other PR where we install + run it on everything.

Reply:
Sounds good. I'll run it on the files that I edit heavily but for everything else, it makes sense to wait and make sure to run it manually in safe mode.